NYCJobsWeb/JobsSearch.cs

```diff
@@ -67,8 +68,8 @@ static JobsSearch()
                 {
                     sp.ScoringProfile = "jobsScoringFeatured";      // Use a scoring profile
                     sp.ScoringParameters = new List<ScoringParameter>();
-                    sp.ScoringParameters.Add(new ScoringParameter("featuredParam", "featured"));
-                    sp.ScoringParameters.Add(new ScoringParameter("mapCenterParam", lon.ToString(CultureInfo.InvariantCulture) + "," + lat.ToString(CultureInfo.InvariantCulture)));
+                    sp.ScoringParameters.Add(new ScoringParameter("featuredParam", new List<string> { "featured" }));
```

Comment:
@liamca Tip: Instead of `new List<string> { "featured" }` you can just write `new[] { "featured" }` and the compiler will infer the element type for you. Arrays and `Lists` both implement `IEnumerable`, so either one works with this constructor.